### PR TITLE
feat: package type dev-alien-headeronly

### PR DESCRIPTION
**Pull request by mhier**
Needed for the Inja package

### CODE REVIEW

File: checkPackageConsistency [hunk 1]
```diff
@@ -215,14 +219,14 @@ for subpackage in ${config[Has_packages]} ; do
     if [ $subpackage != "dynload" ]; then
       # check for presence of -config script
       n_config_files=`dpkg -c $debname | grep "\./usr/bin/.*-config" | wc -l`
```
Comment (killenb) [inline checkPackageConsistency:221]: These are deprecated. Instead, check for pkgconf files.

File: checkPackageConsistency [hunk 2]
```diff
@@ -215,14 +219,14 @@ for subpackage in ${config[Has_packages]} ; do
...
-      if [ "$n_config_files" -ne 1 -a $subpackage != "dev-alien" ]; then
+      if [ "$n_config_files" -ne 1 -a $subpackage != "dev-alien" -a $subpackage != "dev-alien-headeronly" ]; then
         echo "Did not find the config script required for a dev package!"
         ERROR=1
       fi
-    
+
       # check for presence of cmake find package macro
       n_config_files=`dpkg -c $debname | grep "\./usr/share/cmake-.*/Modules/Find.*\.cmake" | wc -l`
```
Comment (killenb) [inline checkPackageConsistency:228]: CMake Find Modules are currently being removed by the renovation by @d-rothe. The normal configs are already checked above. Just remove this check,

File: templates/control.dev-alien-headeronly
Comment (killenb) [inline templates/control.dev-alien-headeronly:1]: Is this a symlink to the normal control.dev?

**Review by killenb — APPROVED** [review]
I think the CMake FindModules Section must be removed in order for the scripts to work with the latest project template. To clarify this I requested another review by @d-rothe

File: checkPackageConsistency (hunk 2 above)
Comment (d-rothe) [reply checkPackageConsistency:228]: Let's first merge to master and then resolve conflicts

File: checkPackageConsistency (hunk 1 above)
Comment (d-rothe) [reply checkPackageConsistency:221]: please leave as is here, otherwise we will have more merge conflicts with my PR

**Review by d-rothe — APPROVED** [review]

**Pull request merged**